backend/app/api/endpoints/users.py

```
@@ -312,6 +312,59 @@ class SearchUsersResponse(BaseModel):
+    name = parts[0].strip()
+    namespace = parts[1].strip() if len(parts) > 1 else "default"
+
+    if not name:
+        return None
+
+    return DefaultTeamConfig(name=name, namespace=namespace)
+
+
+@router.get("/default-teams", response_model=DefaultTeamsResponse)
+async def get_default_teams(
+    current_user: User = Depends(security.get_current_user),
```

Ruff (0.14.11)

Unused function argument: `current_user`

(ARG001)